packages/host/app/components/card-catalog/modal.gts

```diff
@@ -237,6 +253,36 @@ export default class CardCatalogModal extends Component<Signature> {
     });
   }
 
+  get cardUrls() {
```

Comment:
For filtering prerendered search by card url (i.e. user enters a card URL in the search field). This is an array because the prerendered search takes an array of card urls to filter (but in this case there will only be one item in the array)